steering-committee/prioritization-criteria.md

```diff
@@ -0,0 +1,64 @@
+# OWD prioritization criteria
+The [OWD charter](https://github.com/openwebdocs/project/blob/main/charter.md) says _"prioritize based on the needs of the global community of web developers and designers"_. This document describes criteria for open and sponsor/vendor-neutral prioritization and will further guide OWD’s decision-making.
+
+## Criteria
+In no particular order. When deciding about what to prioritize and which opportunity to work on, OWD will take into account:
+
```

Comment:
At this point, would it be good to list all the criteria in a simple bulleted list, just to provide an overview?